.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+## Contributing
+
+As a contributor please ensure you have read over and understand Puppets
+Contributing guidelines, they can be found in the root directory of all Supported Modules.
+
+## Reviewing
+
+As a reviewer please go through the following checklist:
+
+- [ ] I understand what this PR is doing
+- [ ] I can confirm it does what it is supposed to do
+- [ ] The title is reflective of the changes
+- [ ] The code readable
+- [ ] Necessary tests have been added
+- [ ] Functionality documented/updated in the readme or [code](https://puppet.com/docs/puppet/latest/puppet_strings.html)
```

Comment:
```suggestion
- [ ] Functionality documented/updated in the readme and [REFERENCE](https://puppet.com/docs/puppet/latest/puppet_strings.html)
```

Do we have some documentation how/where REFERENCE.md gets updated? Is that part of the release-prep?